hypervisor/src/vm.rs

```diff
@@ -273,15 +273,8 @@ pub trait Vm: Send + Sync + Any {
     /// Creates a new KVM vCPU file descriptor and maps the memory corresponding
     fn create_vcpu(&self, id: u8, vm_ops: Option<Arc<dyn VmOps>>) -> Result<Arc<dyn Vcpu>>;
     #[cfg(target_arch = "aarch64")]
-    fn create_vgic(
-        &self,
-        vcpu_count: u64,
-        dist_addr: u64,
-        dist_size: u64,
-        redist_size: u64,
-        msi_size: u64,
-        nr_irqs: u32,
-    ) -> Result<Arc<Mutex<dyn Vgic>>>;
+    #[allow(clippy::too_many_arguments)]
```

Comment:
Well I guess you don't need that given you've replaced all the parameters with `VgicConfig`.

Reply:
Oops! Left over from a previous iteration of this patch...